core/Dockerfile

```diff
@@ -1,20 +1,11 @@
-FROM python:3.9-slim-buster
```

Comment:
Should we also base bootstrap over it ?

Reply:
I'm considering this.. bootstrap will take longer to download. but as it will have the base layer, core will download faster the first time.

I guess we can do it for consistency.

Reply:
Good point, maybe base over the same slim should help the 3.9 image.